alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/AbstractJobResourceActions.java

```diff
@@ -194,6 +194,8 @@ public final ValidationActionResponse test(JobFieldModel resource) {
             return ValidationActionResponse.createOKResponseWithContent(validationResponse);
         }
         ValidationActionResponse response = testWithoutChecks(resource);
+        // FIXME this should only return OK if validation COMPLETED successfully (even
+        //  if there are field errors), not if there were other types of errors unrelated to validation
```

Comment:
Im not sure what this comment means, I found an issue with 6.3.0 where the test discovers a certificate error but it does not make it to the UI because this is changing the response to an OK status. Please refer to https://github.com/blackducksoftware/blackduck-alert/pull/1289 to see the changes I would like to make in this area.

Reply:
I created a 6.5.0 Jira issue for this as well. I'll take a look at your PR.

Reply:
Your issue is different. This is addressing the fact that field-errors produce a 2xx response (which they should because the validation _completed_ successfully), but errors during validation also result in a 2xx which they should not.